### DISCUSSION TITLE
small fix props in RadioGroup

### DISCUSSION THREAD

**Discussion by DaveMaks** [skeletonlabs/skeleton#3002] [Ideas]
Hi everyone! 

https://github.com/skeletonlabs/skeleton/blob/9344fd6f6bf588d306b2b2fc6d32eb606e1db536/packages/skeleton/src/lib/components/Radio/RadioGroup.svelte#L52 

I think it is necessary to correct this line and replace `${cBase}` with `${padding?padding:cBase}`,  because padding is not used anywhere

**Comment by endigo9740** [comment]
@DaveMaks so padding isn't used in this (the parent) component. It's passed down to the children components via the Context API. The benefit of setting this on the parent was so it can be done once and match for all children.
